chore: Update security issue notification (#237)
aws/aws-encryption-sdk-python: CONTRIBUTING.md

information to effectively respond to your bug report or contribution.

## Security issue notifications

> If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.

Reviewer: ```suggestion
If you discover a potential security issue in this project, notify AWS/Amazon Security by using our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public GitHub issue.
```
